test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,96 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding with Strimzi Kafka Cluster database without applicationSelector
```

Comment:
I do not see the correlation between the issue we are fixing in this PR and the added tests. Namely, we are fixing a bug, and the added acceptance test scenarios should verify if the bug is still present or not, so that we can catch possible regressions later on. From the discussion, I could derive a following scenario that needs to be tested:

* backing service status is created initially with no credentials, i.e.empty 
* as the result of that, SBO bind empty credentials to the application
* backing service status gets updated, containing now credentials
* SBO gets notified and it reinject the proper credentials into the application

Is my observation correct? The current scenario does not speak about the above.

The current scenario does not focus on this.

Reply:
Added a new set of acceptance test wherein the value of the custom env var is checked